package/doc/sphinx/source/conf.py

```diff
@@ -133,7 +133,7 @@
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
-# styles/fonts to match http://mdanalysis.org (see public/css)
+# styles/fonts to match https://mdanalysis.org (see public/css)
```

Comment:
Won't this change again with https://github.com/MDAnalysis/MDAnalysis.github.io/issues/135 ?

Reply:
For right now, I'll use the User Guide as style reference as suggested. We should have a style guide page with our design colors etc, though, so that we can quickly reference it.

Reply:
Added https://github.com/MDAnalysis/mdanalysis/wiki/MDAnalysis-theme-colours